processing/data_collection/gazette/spiders/ap_macapa.py

```diff
@@ -0,0 +1,79 @@
+import datetime as dt
+
+import dateparser
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class ApMacapaSpider(BaseGazetteSpider):
+
+    name = "ap_macapa"
+    allowed_domains = ["macapa.ap.gov.br"]
+    start_date = None
+
+    TERRITORY_ID = "1600303"
+
+    def __init__(self, start_date=None, end_date=None, *args, **kwargs):
+        if start_date is None:
+            # Start from yesterday
+            start_date = dt.date.today() + dt.timedelta(days=-1)
```

Comment:
I think this way is more readable: 

```suggestion
            start_date = dt.date.today() - dt.timedelta(days=1)
```

Reply:
Removed that. No longer applicable.